cts/Makefile.am

```diff
@@ -42,13 +42,18 @@ ctslib_PYTHON	=	__init__.py		\
 			remote.py		\
 			watcher.py
 
-cts_DATA	=	README.md cts.supp
+cts_DATA	=	README.md cts.supp pacemaker-cts-dummyd.service
+
+if BUILD_UPSTART
+cts_DATA	+=	pacemaker-cts-dummyd.conf
+endif
```

Comment:
1.1 doesn't need pacemaker-cts-dummyd.conf, so we can just drop that part here; in 2.0, it's used by cts-exec